src/main/java/taxiservice/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,238 @@
+package taxiservice.dao.jdbc;
+
+import static taxiservice.dao.jdbc.DriverDaoJdbcImpl.DRIVERS_TAB_NAME;
+import static taxiservice.dao.jdbc.ManufacturerDaoJdbcImpl.MANUFACTURERS_TAB_NAME;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxiservice.dao.CarDao;
+import taxiservice.exception.DataProcessingException;
+import taxiservice.lib.Dao;
+import taxiservice.model.Car;
+import taxiservice.model.Driver;
+import taxiservice.model.Manufacturer;
+import taxiservice.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbcImpl implements CarDao {
+    public static final String CARS_TAB_NAME = "cars";
+    public static final String CARS_DRIVERS_TAB_NAME = "cars_drivers";
```

Comment:
```suggestion
    public static final String CARS_DRIVERS_TAB_NAME = "cars_drivers";
```
I not sure, but I think this is bad practice

Reply:
agree,  let's write query as it is.